docs/openj9_support.md

```diff
@@ -66,60 +65,147 @@ For any issues or limitations of an Eclipse OpenJ9 release, read the [release no
+| Windows                    |  x32   |  x64  |
+|----------------------------|--------|-------|
+| Windows 10                 |   Y    |   Y   |
+| Windows Server 2012        |   Y    |   Y   |
+| Windows Server 2012 R2     |   Y    |   Y   |
+| Windows Server 2016        |   Y    |   Y   |
 
-OpenJDK8 binaries are supported on the minimum operating system levels shown in the table.
-For Linux, other distributions are supported provided that they have a Linux kernel v3.2 or later
-(Look in the `/proc/version` file to find the kernel version number).
+
+| AIX          |  ppc32   |  ppc64   |
+|--------------|----------|----------|
+| AIX 7.1 TL4  |    Y     |    Y     |
+| AIX 7.2      |    Y     |    Y     |
 
 When public support for an operating system version ends, OpenJ9 can no longer be supported on that level.
 
-### OpenJDK 9 build environment
 
-| Platform                    | Operating system         |  Compiler                       |   Comments     |
-|-----------------------------|--------------------------|---------------------------------|----------------|
-| Linux x86 64-bit            | Ubuntu 16.04 (kernel 4.4)| gcc 4.8.5                       |                |
-| Linux on POWER LE 64-bit    | Ubuntu 16.04             | gcc 4.8.5                       |                |
-| Linux on IBM Z 64-bit       | Ubuntu 16.04             | gcc 4.8.5                       |                |
-| Windows x86 64-bit          | Windows Server 2016      | Microsoft Visual Studio 2013    |                |
-| AIX POWER BE 64-bit         | AIX 7.1 TL04             | xlc/C++ 13.1.3                  |                |
+### OpenJDK 9
+
+OpenJDK9 binaries are supported on the minimum operating system levels shown in the following tables:
+
+
+| Linux                                 |  x32   |  x64   |   ppc64le  | Z31  | Z64  |
+|---------------------------------------|--------|--------|------------|------|------|
+| Centos 6                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Centos 7                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Red Hat Enterprise Linux (RHEL) 6     |   Y    |   Y    |     Y      |   Y  |  Y   |
+| RHEL 7                                |   Y    |   Y    |     Y      |   Y  |  Y   |
+| SUSE Linux Enterprise Server (SLES) 12|   Y    |   Y    |     Y      |   Y  |  Y   |
+| Ubuntu 16.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+| Ubuntu 18.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i><span class="sr-only">Note</span> **Note:** Not all of these distributions are tested, but Linux distributions that have a
+minimum glibc version 2.12 are expected to function without problems.
+
+| Windows                    |  x32   |  x64  |
+|----------------------------|--------|-------|
+| Windows 10                 |   Y    |   Y   |
```

Comment:
32 bit windows was only intended to be supported for JDK8. No effort has been put into making it function for new JDK levels